### ISSUE TITLE
Invalid JSON input error when trying to deploy an app

### ISSUE THREAD

**Issue by lmeninato** [rstudio/shiny#2273]
I'm not sure if the error is from the way my environment is set up, but I can no longer publish an app I've been working on for a while. I think the error might have been caused when I was handling some merge conflicts, but the only way for me to deploy an update is to delete the dcf file and republish in a NEW location. This is frustrating because I have to let all the users know that the app's url has changed. Below you can see the error I get. 
![invalid_json_rshiny_error](https://user-images.githubusercontent.com/8249180/49099623-f27deb00-f23f-11e8-8e63-967e808008dc.PNG)

**Comment by colearendt** [comment]
Do you still have the `dcf` file for us to look at?

**Comment by lmeninato** [comment]
name: delorean-test
title: delorean-test
username: Lorenzo.Meninato@ny.frb.org
account: Lorenzo.Meninato@ny.frb.org
server: j1rconnect01.kc.frbres.org
hostUrl: https://j1rconnect01.kc.frbres.org/__api__
appId: 114
bundleId: 416
url: https://j1rconnect01.kc.frbres.org/content/114/
when: 1542131323.34035
asMultiple: FALSE
asStatic: FALSE
ignoredFiles: .RData|README.md|test/test_mars.R|test/test.R

**Comment by colearendt** [comment]
Thanks for sharing that! Let me do some poking on my end. This is probably an issue more relevant for github.com/rstudio/rsconnect . One thing I would recommend is to use [vanity URLs](http://docs.rstudio.com/connect/admin/content-management.html#vanity-paths) if you can, as it makes switching location on the server a bit easier. The user only has to worry about a nice, human-readable path. At present, it is unfortunately a feature only permitted for administrators.

**Comment by lmeninato** [comment]
Good idea to ask around on rsconnect. I'll try and pester my admins to let me get a vanity url.
